src/box/txn_limbo.h

```diff
@@ -474,6 +474,10 @@ txn_limbo_fence(struct txn_limbo *limbo);
 void
 txn_limbo_unfence(struct txn_limbo *limbo);
 
+/** Return whether limbo has an owner. */
+bool
+txn_limbo_has_owner(struct txn_limbo *limbo);
```

Comment:
This requires adding the `replication.h` header for the `REPLICA_ID_NIL` definition, while the `snap_quorum_delay.cc` test defines its own `instance_id`, which causes a compilation error:
https://github.com/tarantool/tarantool/blob/b0b32bff022caeffd64ca687baab908f25fa2fcb/test/unit/snap_quorum_delay.cc#L76-L82

However, according to 8c50e069:
> Also the commit disables snap_quorum_delay.test, which uses internal API of replication and txn modules assuming the journal is initialized somewhere inside. But now it is not, and it can't be fixed in a sane way inside the test. It will be deleted/rewritten later.

So I guess it safe to simply remove the `instance_id` definition from `snap_quorum_delay.cc`.

Reply:
Oh. So we have a test disabled for 4 years, huh? :)